Resources/Private/Partials/Columns/Select.html

```
@@ -1,22 +1,29 @@
-        </xm:link.editRecord>
-    </f:then>
-    <f:else>
-        <span class="workspace-state-{item.state}">{fieldValue}</span>
-    </f:else>
-</f:if>
+<span class="badge-container select-badge-container">
+    <f:if condition="{item._{colConfig.columnName}}">
+        <f:for each="{item._{colConfig.columnName}}" as="relation">
+            <f:if condition="{item.editable}">
+                <f:then>
+                    <xm:link.editRecord
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)